apps/src/gamelab/GameLabP5.js

```diff
@@ -339,6 +339,26 @@ GameLabP5.prototype.init = function (options) {
     return this;
   };
 
+  // Save the original implementation to allow for optional parameters.
+  if (!window.p5.prototype.originalEllipse_) {
+    window.p5.prototype.originalEllipse_ = window.p5.prototype.ellipse;
```

Comment:
If it does need to live here (perhaps for testing purposes?) it might be clearer to reference it the same way here and line 348 (same for originalRect)